code/_helpers/game.dm

```diff
@@ -449,7 +449,7 @@
 /proc/SecondsToTicks(var/seconds)
 	return seconds * 10
 
-/proc/round_is_spooky(var/spookiness_threshold = config.cult_ghostwriter_req_cultists)
+/proc/round_is_spooky(var/spookiness_threshold = get_config_value(/decl/config/num/cult_ghostwriter_req_cultists))
```

Comment:
Shouldn't this config option be renamed to something a bit more meaningful, since it impacts more things than just ghost blood writing?

Reply:
Yeah, that makes sense.